budget-vs-actual/provincial/pipeline-spec.yaml

```diff
@@ -196,46 +514,62 @@ budget-vs-actual-provincial:
     - run: add_computed_field
       parameters:
         resources:
+          - epre-2019-20
+          - epre-2018-19
           - epre-2017-18
           - epre-2016-17
           - epre-2015-16
+          - are-2016-17
+          - are-2015-16
         fields:
         - operation: format
           target: corrected_econ2
-          with: '{economic_classification_1}'
+          with: '{EconomicClassification1}'
```

Comment:
I don't think this is correct - the point of this corrected_econ stuff was because in previous sources what they called economic_classification_1 was actually what should end up as EconomicClassification2. Unless I got things confused, the datasets where we have a field `EconomicClassification`_n_ are correct, and `EconomicClassification` _n_ and  `corrected_econ`_n_ just need to be pulled together to a column named `EconomicClassification`_n_ in the join at the end.

Reply:
Oh, I see now. Actually, I couldn't figure it out what it is for. I'm deleting them.

Reply:
Corrected_econs deleted since they are not necessary.

https://github.com/vulekamali/treasury-pipelines/pull/26/commits/ed4afb10e1d0ffa0442da44e63cf332959d84955